pkg/R/utils.R

```diff
@@ -218,3 +218,53 @@ sortKeyValueList <- function(kv_list) {
   keys <- sapply(kv_list, function(x) x[[1]])
   kv_list[order(keys)]
 }
+
+## Utility function to generate compact R lists from grouped rdd 
```

Comment:
minor: follow the existing style using one # and one following space; similar below